keylime/migrations/versions/a72aec03d720_migrate_allowlists_to_separate_table.py

```diff
@@ -0,0 +1,77 @@
+"""Migrate allowlists to dedicated table (1/2)
+
+Revision ID: a72aec03d720
+Revises: bc3b6b551b0a
+Create Date: 2022-07-21 12:17:17.779159
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a72aec03d720'
+down_revision = 'bf48e0c4751d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()[f"upgrade_{engine_name}"]()
+
+
+def downgrade(engine_name):
+    globals()[f"downgrade_{engine_name}"]()
+
+
+
+
+
+def upgrade_registrar():
+    pass
+
+
+def downgrade_registrar():
+    pass
+
+
+def upgrade_cloud_verifier():
+
+    # Migrate existing agent info to the allowlists table.
+    conn = op.get_bind()
+
+    res = conn.execute("SELECT agent_id, tpm_policy, allowlist FROM verifiermain")
```

Comment:
I thought we already have an `allowlist` table. Is data still populated in the `verifiermain` table for the allowlists? Is this a leftover from the partial allowlist API functionality from before?

Reply:
To clarify: this is grabbing the `allowlist` **column** present within the `verifiermain` **table**. The migration then crafts it into an object that is then bulk inserted into the separate `allowlists` **table**.